web-registry/src/lib/queries/react-query/ecocredit/basket/getBasketBalances/getBasketBalancesQuery.ts

```diff
@@ -0,0 +1,26 @@
+import {
+  ReactQueryBasketBalancesProps,
+  ReactQueryBasketBalancesResponse,
+} from './getBasketBalancesQuery.types';
+import { getBasketBalancesQueryKey } from './getBasketBalancesQuery.utils';
+
+export const getBasketBalancesQuery = ({
+  client,
```

Comment:
do we have multiple `clients`? I was a bit curious why we weren't consuming RQ's `useQuery` directly in these hooks and are instead passing the return of these to `useQuery`... it seems like being able to be flexible with `client` is the main reason?

Reply:
We do because those are not RQ client but ledger clients (ecocredit, basket, bank ...). Reasons to extract queries are:
- reusability of queries (main point)
- being able to access the key of an instantiated query to invalidate it
- also some queries might have more logic than just a call making them hard to read if directly consumed by hooks.